README.md

```diff
@@ -37,6 +37,209 @@
+
+		var textDescription: String {
+			return "Name: \(self.name) \nHeight: \(self.height) \nFavorite Food: \(self.favoriteFood)"
+		}
+ 
+## Create an Instance of the Person Class
+Now, you can create an instance of the class, and it will have the `textDescription` variable created automatically from the values you supply for the class. Here’s how you create an instance of the class, and then print `textDescription`.
+
+		let jim = Person(name: "Jim", height: 6.0, favoriteFood: “Pasta")
+		print (jim.description)
+
+Here’s the result
+
+		// Name: Jim Height: 6.0 Favorite Food: Pasta
+
+## Generalizing the Person Class
+You can use this as a model for all sorts of other classes: Dog, Cat, Building, Car, or anything you want. Just declare the class, declare the relevant variables, build the initializer, and create a `textDescription` variable that you lets you print out the data inside the class.
+
+Of course, the actual properties inside these other classes will vary. For example, if you create a Building class, here’s what you might need. Instead of name, you might need address. Height will be ok — both people and buildings have height. Buildings don’t have favorite foods, so you might just omit that. Here’s what you get when you create a Building class with address and height:
+
+		class Building {
+
+    			// here are the properties
+    			var address: String
+    			var height: Double
+			var textDescription: String {
+        			return "Address: \(self.address) \nHeight: \(self.height)"
+    			}
+
+    			// here is the initializer method
+    			init(address: String, height: Double) {
+        			address = name
+        			self.height = height
+   			}
+		}
+		
+Here is an example of instantiating an instance of Building (a library in this case).
+
+		let library = Building(address: “Main Street", height: 20.0, favoriteFood: "Pasta")
+
+		print(description)
+			//Address: Main Street
+			//Height: 20.0
+
+This patterns lets you create all sorts of classes for all sorts of objects. You simply declare the relevant properties and set them in the `init` method. As long as you remember to always implement the code for textDescription, you’ll always be able to get a text version of the data in any of your class instances.
+
+## Customizing textDescription for Each Class with a Protocol
+This is the declaration of `textDescription` that you have to customize each time, but once you do that, you’ll always have it ready when you need to print it out or display it. You just have to remember to declare `textDescription`.
+
+You don’t have to make mental notes or keep a list of reminders for each class you create: you can use a *protocol* to specify that your class must be able to return a `textDescription` property. You still have to write the specific `textDescription` code in each class, but once you’re done that, you can always write
+
+		print (myDog.textDescription)
+		print (myBuilding.textDescription)
```

Comment:
Me being nit-picky (sorry!).

I would prefer there to be no space between `print` and the `( )` which contain the values to be printed, like so:

```swift
print(myDog.textDescription)
print(myBuilding.textDescription)
```